### ISSUE TITLE
Imported font management (install and uninstall from system)

### ISSUE THREAD

**Issue by eirinn1975** [character-map-uwp/Character-Map-UWP#173]
Hello,
Is there a way to effectively install the imported font to the system with the app?
If yes, what's the procedure?

**Comment by JohnnyWestlake** [comment]
Not currently, due to sandboxing limitations with the way font installing works on Windows. If you have the original font files, you can open it in the default font view and install from there (there should be an install button in the top right). Otherwise, you can Export the font (Ctrl + S) to a file, then open the file in the default Windows Font Viewer and install from there. 

I may be able to add a way to open the default font viewer from within the app, but there's no way to automate the installation right now.

**Comment by eirinn1975** [comment]
Yep, I didn't think of that limitation. I already did as you said. :)
